db/system_keyspace.cc

```diff
@@ -233,12 +233,15 @@ schema_ptr system_keyspace::topology() {
             .with_column("request_id", timeuuid_type)
             .with_column("ignore_nodes", set_type_impl::get_instance(uuid_type, true), column_kind::static_column)
             .with_column("new_cdc_generation_data_uuid", timeuuid_type, column_kind::static_column)
+            .with_column("new_keyspace_rf_change_ks_name", utf8_type, column_kind::static_column)
+            .with_column("new_keyspace_rf_change_data", map_type_impl::get_instance(utf8_type, utf8_type, false), column_kind::static_column)
             .with_column("version", long_type, column_kind::static_column)
             .with_column("fence_version", long_type, column_kind::static_column)
             .with_column("transition_state", utf8_type, column_kind::static_column)
             .with_column("committed_cdc_generations", set_type_impl::get_instance(cdc_generation_ts_id_type, true), column_kind::static_column)
             .with_column("unpublished_cdc_generations", set_type_impl::get_instance(cdc_generation_ts_id_type, true), column_kind::static_column)
             .with_column("global_topology_request", utf8_type, column_kind::static_column)
+            .with_column("global_topology_request_id", utf8_type, column_kind::static_column)
```

Comment:
Should be `timeuuid_type`. Same as non global `request_id` since they serve the same purpose.

Reply:
_Should_ or _can_ be? I mean, it works as is, furthermore - it's just a generic uuid, not necessarily time-based.

Reply:
Should. Correct types are important. Otherwise why do we have schema at all?

Reply:
And this is not something that can be fixed in a cleanup afterwords if released this way.

Reply:
Right, thanks. I changed the column to the `timeuuid` type.